CONTRIBUTORS

```diff
@@ -37,6 +37,7 @@ Thank you!
     Amos Jeffries <squid3@treenet.co.nz>
     Amos Jeffries <yadij@users.noreply.github.com>
     Anatoli <me@anatoli.ws>
+    Andre Albsmeier <45147422+aafbsd@users.noreply.github.com>
     Andre Albsmeier <Andre.Albsmeier@siemens.com>
```

Comment:
@aafbsd, I have added you to CONTRIBUTORS using a combination of your Squid Bugzilla account name and GitHub email. Do _both_ of the above same-name entries correspond to you? If yes, do you want to keep both (with different email addresses[^1]) or merge them into one?

[^1]: As discussed [earlier](https://github.com/squid-cache/squid/pull/2100#discussion_r2166856278), you can still change the email address in the new entry by, for example, making your GitHub email public.

Reply:
OK, I will clear this PR for merging as is then and submit a followup PR that removes the duplicate entry added in this PR (I will ping you in case you want to replace/update that old Siemens entry at that time or add a new one). 

AFAICT, the above is the easiest way to work around our CI check that insists on your current GitHub credentials to be present in CONTRIBUTORS. We need to enhance that check to be able to overwrite GitHub-generated metadata more easily...